index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
Does this index.html need to be checked in? we can prob leave it in the gitignore as its more of a dev dep

Reply:
True. You'll just have to make sure it's in the folder yourself